src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java

```diff
@@ -75,7 +74,7 @@ public void execute_duplicateInternship_throwsCommandException() {
 
     @Test
     public void execute_newNote_success() {
-        Note validNote = TypicalNote.FINISH_HOMEWORK;
+        Note validNote = TypicalNote.FINISH_RESUME_2;
```

Comment:
Why did we have to change names for all of these?

Reply:
Because right now in the SampleResumeBook for model, for each Item I have already put 2 instances in the list to test for duplicate. So I created a third one and use it here for successful edit first so that later I can test for duplicate as well, using 3 internships, 3 projects,...